Backpack/src/androidTest/java/net/skyscanner/backpack/badge/BpkBadgeTest.kt

```diff
@@ -53,17 +55,16 @@ class BpkBadgeTest {
   }
 
   @Test
-  @Ignore // FIXME: Test is not working anymore (the component looks fine)
   fun test_alpha_default() {
     val badge = BpkBadge(context)
     badge.type = BpkBadge.Type.Success
-    Assert.assertEquals(PixelFormat.OPAQUE, badge.background.opacity)
+    Assert.assertEquals(context.getColorStateList(R.color.bpkGlencoe), (badge.background as GradientDrawable).color)
```

Comment:
the background was _not_ opaque in the way the system treats it, but the main colour of the background was not transparent (unlike the outline badge test)